src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/CaseUpdaterTest.java

```diff
@@ -330,4 +130,74 @@ public void should_not_update_case_data_without_any_documents() {
         assertThat(exception).hasMessage("Missing scanned documents in exception record");
         verifyNoInteractions(addressExtractor);
     }
+
+    private void assertAddress(Address resultCaseAddress, Address exceptionRecordAddress) {
+        assertThat(resultCaseAddress.addressLine1).isEqualTo(exceptionRecordAddress.addressLine1);
+        assertThat(resultCaseAddress.addressLine2).isEqualTo(exceptionRecordAddress.addressLine2);
+        assertThat(resultCaseAddress.addressLine3).isEqualTo(exceptionRecordAddress.addressLine3);
+        assertThat(resultCaseAddress.country).isEqualTo(exceptionRecordAddress.country);
+        assertThat(resultCaseAddress.county).isEqualTo(exceptionRecordAddress.county);
+        assertThat(resultCaseAddress.postCode).isEqualTo(exceptionRecordAddress.postCode);
+        assertThat(resultCaseAddress.postTown).isEqualTo(exceptionRecordAddress.postTown);
+    }
+
+    private void assertCaseData(SampleCase resultCaseData, SampleCase originalCase) {
+        assertThat(resultCaseData)
+            .extracting(c -> tuple(
+                c.legacyId,
+                c.firstName,
+                c.lastName,
+                c.dateOfBirth,
+                c.bulkScanCaseReference
+            ))
+            .isEqualTo(tuple(
+                originalCase.legacyId,
+                originalCase.firstName,
+                originalCase.lastName,
+                originalCase.dateOfBirth,
+                originalCase.bulkScanCaseReference
+            ));
+    }
+
+    private void assertScannedDocuments(
```

Comment:
only used once, and it hide's what's being asserted

Reply:
It will be used in new methods. This PR is to make the next PR smaller.